cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -426,32 +426,6 @@ impl xcm_executor::Config for XcmConfig {
 			ResolveAssetTo<StakingPot, crate::NativeAndNonPoolAssets>,
 			AccountId,
 		>,
-		// This trader allows to pay with `is_sufficient=true` "Trust Backed" assets from dedicated
-		// `pallet_assets` instance - `Assets`.
-		cumulus_primitives_utility::TakeFirstAssetTrader<
-			AccountId,
-			AssetFeeAsExistentialDepositMultiplierFeeCharger,
```

Comment:
Some of these helper types are not used since you remove this trader, please remove them too.
e.g. `AssetFeeAsExistentialDepositMultiplierFeeCharger`, `XcmAssetFeesReceiver`, `ForeignAssetFeeAsExistentialDepositMultiplierFeeCharger`